<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>SlicerDWD</class>
  <widget class="qMRMLWidget" name="SlicerDWD">
    <layout class="QVBoxLayout">
      <item>
        <widget class="ctkCollapsibleButton">
          <property name="text">
            <string>Inputs</string>
          </property>
          <layout class="QFormLayout">
            <item row="0" column="0">
              <widget class="QLabel">
                <property name="text">
                  <string>Training Dataset</string>
                </property>
              </widget>
            </item>
            <item row="0" column="1">
              <widget class="ctkPathLineEdit" name="pathTrain">
              </widget>
            </item>
            <item row="1" column="0">
              <widget class="QCheckBox" name="chkSample">
                <property name="text">
                  <string>Split Training and Testing Data</string>
                </property>
                <property name="checked">
                  <bool>true</bool>
                </property>
              </widget>
            </item>
            <item row="2" column="0">
              <widget class="QLabel" name="spnSampleLabel">
                <property name="text">
                  <string>Testing Data Sample Size (%)</string>
                </property>
              </widget>
            </item>
            <item row="2" column="1">
              <widget class="QSpinBox" name="spnSample">
                <property name="minimum">
                  <number>0</number>
                </property>
                <property name="maximum">
                  <number>100</number>
                </property>
                <property name="singleStep">
                  <number>5</number>
                </property>
                <property name="value">
                  <number>30</number>
                </property>
              </widget>
            </item>
            <item row="3" column="0">
              <widget class="QLabel" name="pathTestLabel">
                <property name="text">
                  <string>Testing Dataset</string>
                </property>
                <property name="enabled">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="3" column="1">
              <widget class="ctkPathLineEdit" name="pathTest">
                <property name="enabled">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="4" column="0">
              <widget class="QLabel">
                <property name="text">
                  <string>Additional Metrics</string>
                </property>
              </widget>
            </item>
            <item row="4" column="1">
              <widget class="ctkPathLineEdit" name="pathMetrics">
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="ctkCollapsibleButton">
          <property name="text">
            <string>Training</string>
          </property>
          <layout class="QFormLayout">
            <item row="0" column="0" colspan="2">
              <widget class="QCheckBox" name="chkAutoTune">
                <property name="text">
                  <string>Automatic Tuning</string>
                </property>
                <property name="checked">
                  <bool>true</bool>
                </property>
              </widget>
            </item>
            <item row="1" column="0">
              <widget class="QLabel" name="spnTuningCLabel">
                <property name="text">
                  <string>Tuning Parameter (C)</string>
                </property>
                <property name="enabled">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="spnTuningC">
                <property name="enabled">
                  <bool>false</bool>
                </property>
                <property name="decimals">
                  <number>5</number>
                </property>
                <property name="minimum">
                  <double>0.000000000000000</double>
                </property>
                <property name="maximum">
                  <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                  <double>0.001000000000000</double>
                </property>
                <property name="value">
                  <double>0.002500000000000</double>
                </property>
              </widget>
            </item>
            <item row="2" column="0" colspan="2">
              <widget class="QPushButton" name="btnTrain">
                <property name="text">
                  <string>Train Classifier</string>
                </property>
                <property name="enabled">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <widget class="ctkCollapsibleButton">
          <property name="text">
            <string>Testing</string>
          </property>
          <layout class="QFormLayout">
            <item row="0" column="0">
              <widget class="QCheckBox" name="chkSaveResults">
                <property name="text">
                  <string>Save Test Results</string>
                </property>
                <property name="checked">
                  <bool>true</bool>
                </property>
              </widget>
            </item>
            <item row="1" column="0">
              <widget class="QLabel" name="pathResultsLabel">
                <property name="text">
                  <string>Test Results</string>
                </property>
              </widget>
            </item>
            <item row="1" column="1">
              <widget class="ctkPathLineEdit" name="pathResults"/>
            </item>
            <item row="2" column="0">
              <widget class="QLabel">
                <property name="text">
                  <string>Testing Statistics</string>
                </property>
                <property name="visible">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="3" column="0" colspan="2">
              <widget class="QTableWidget" name="tblStats">
                <property name="editTriggers">
                  <set>QAbstractItemView::NoEditTriggers</set>
                </property>
                <property name="rowCount">
                  <number>2</number>
                </property>
                <property name="columnCount">
                  <number>3</number>
                </property>
                <attribute name="horizontalHeaderVisible">
                  <bool>false</bool>
                </attribute>
                <attribute name="verticalHeaderVisible">
                  <bool>false</bool>
                </attribute>
              </widget>
            </item>
            <item row="4" column="0" colspan="2">
              <widget class="QPushButton" name="btnTest">
                <property name="text">
                  <string>Test Classifier</string>
                </property>
                <property name="enabled">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="5" column="0" colspan="2">
              <widget class="QPushButton" name="btnMean">
                <property name="text">
                  <string>Compute Projected Mean Shape</string>
                </property>
                <property name="enabled">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="6" column="0" colspan="2">
              <widget class="QPushButton" name="btnKDE">
                <property name="text">
                  <string>Compute KDE</string>
                </property>
                <property name="enabled">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="8" column="0">
              <widget class="QPushButton" name="btnCorr">
                <property name="text">
                  <string>Correlate with</string>
                </property>
                <property name="enabled">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
            <item row="8" column="1">
              <widget class="QComboBox" name="comCorr">
                <property name="enabled">
                  <bool>false</bool>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>
      <item>
        <spacer name="verticalSpacer">
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
            <size>
              <width>20</width>
              <height>40</height>
            </size>
          </property>
        </spacer>
      </item>
    </layout>
  </widget>
  <customwidgets>
    <customwidget>
      <class>ctkCollapsibleButton</class>
      <extends>QWidget</extends>
      <header>ctkCollapsibleButton.h</header>
      <container>1</container>
    </customwidget>
    <customwidget>
      <class>ctkPathLineEdit</class>
      <extends>QWidget</extends>
      <header>ctkPathLineEdit.h</header>
    </customwidget>
    <customwidget>
      <class>ctkSliderWidget</class>
      <extends>QWidget</extends>
      <header>ctkSliderWidget.h</header>
    </customwidget>
    <customwidget>
      <class>qMRMLNodeComboBox</class>
      <extends>QWidget</extends>
      <header>qMRMLNodeComboBox.h</header>
    </customwidget>
    <customwidget>
      <class>qMRMLWidget</class>
      <extends>QWidget</extends>
      <header>qMRMLWidget.h</header>
      <container>1</container>
    </customwidget>
  </customwidgets>
  <resources/>
  <connections>
    <!--    <connection>-->
    <!--      <sender>DWD</sender>-->
    <!--      <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>-->
    <!--      <receiver>inputSelector</receiver>-->
    <!--      <slot>setMRMLScene(vtkMRMLScene*)</slot>-->
    <!--    </connection>-->
    <connection>
      <sender>DWD</sender>
      <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
      <receiver>outputSelector</receiver>
      <slot>setMRMLScene(vtkMRMLScene*)</slot>
    </connection>
    <!--    <connection>-->
    <!--      <sender>DWD</sender>-->
    <!--      <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>-->
    <!--      <receiver>invertedOutputSelector</receiver>-->
    <!--      <slot>setMRMLScene(vtkMRMLScene*)</slot>-->
    <!--    </connection>-->
  </connections>
</ui>
